lib/grab-project.js

```diff
@@ -24,18 +31,23 @@ export async function grabProject(context) {
       `${context.module.name} ${packageManager}:`,
       `Downloading project: ${packageName}`
     );
+
+    // Default timeout to 10 minutes if not provided.
+    const timeout = context.options.timeout || 1000 * 60 * 10;
+
+    if (packageName.startsWith('http:') || packageName.startsWith('https://')) {
```

Comment:
why a double slash on one but not the other?

Reply:
I don't know, a mistake. Which version would you suggest?

Reply:
not sure it matters; I’d probably do `/^https?:\/\//.test()` personally but i don’t have any concrete argument why that’s better. i was just curious about the inconsistency.